processor/spanmetricsprocessor/processor.go

```diff
@@ -32,6 +32,7 @@ import (
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	"go.opentelemetry.io/collector/pdata/ptrace"
 	conventions "go.opentelemetry.io/collector/semconv/v1.6.1"
+	"go.uber.org/atomic"
```

Comment:
I would rather revert the offending PR than do this change, do you know which one failed?